From 87ad49c703212a8a386d9011aea7e48ede272683 Mon Sep 17 00:00:00 2001
From: Harmen Stoppels <harmenstoppels@gmail.com>
Date: Fri, 14 Aug 2020 14:03:53 +0200
Subject: [PATCH] Define the Tensile_LIBRARY_FORMAT variable

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6f68fe47..392c81f7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -167,6 +167,7 @@ if( BUILD_WITH_TENSILE )
   set( Tensile_ARCHITECTURE "all" CACHE STRING "Tensile to use which architecture?")
   set( Tensile_CODE_OBJECT_VERSION "V2" CACHE STRING "Tensile code_object_version")
   set( Tensile_COMPILER "hcc" CACHE STRING "Tensile compiler")
+  set( Tensile_LIBRARY_FORMAT "msgpack" CACHE STRING "Tensile library format")
 
   option( Tensile_MERGE_FILES "Tensile to merge kernels and solutions files?" ON )
   option( Tensile_SHORT_FILENAMES "Tensile to use short file names? Use if compiler complains they're too long." OFF )
-- 
2.25.1

